src/pages/documentation/mobile-core/signal/rules-engine-integration.md

```diff
@@ -0,0 +1,105 @@
+# Signal extension and Rules Engine integration
+
+The rules that you set up can use the available triggers and conditions, which result in one of the following actions:
+
+* Send a postback
+* Send Personally Identifiable Information (PII)
+* Open a URL
+
+After these actions have been configured to be triggered and published, the Signal extension carries out the requested actions.
+
+To send PII data to external destinations, the **Send PII** action can trigger the rules engine when certain triggers and traits match. When setting a rule, you can also set the **Send PII** action for a Signal event. The `collectPii` API can then be used to trigger the rule and send the data.
+
+## Rules tokens
+
+Rules tokens are special strings that are used in rule actions as values and are expanded by the SDK when the action is carried out. The format of a token is `{%%token%%}`, where `token` is any data element that is defined in the Data Collection UI for a mobile property that identifies the source of the data from which the token is expanded. For example, `{%%My Data element for ECID%%}` can be used in the **Signal Postback** action, where `My Data element for ECID` is a data element that was created using the Mobile Core extension, and the data element type is **Experience Cloud ID**.
+
+The token can also be one of the reserved key names. For more information, see the [matching and retrieving values by keys tutorial](../rules-engine/technical-details.md#matching-and-retrieving-values-by-keys).
+
+Some tokens are modifier functions that specify the transformation that is applied to the value that was replaced by the token. An example is `urlenc`, which specifies that the value will be URL-encoded before it is replaced in the rule.
+
+### Using tokens in Postbacks and PII rule actions
+
+The **Send Postback** and **Send PII** actions allow you to specify a `URL` field and an optional `Post Body` field. You can specify which tokens should be expanded by the Experience Platform SDKs when the postback or PII network call is triggered. For more information on tokens, see the [rule tokens documentation](#rules-tokens).
```

Comment:
Since the rule tokens section is within this very doc, I'd go with "... see the [rule tokens section]".